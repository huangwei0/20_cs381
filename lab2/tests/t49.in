debug off
stopwords off
length 10
load tracks.txt
preprocess
sequence howdy
sequence hello
sequence hola
sequence gutentag
sequence ciao
sequence bonjour
sequence ola
sequence hej