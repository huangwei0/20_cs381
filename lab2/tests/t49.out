howdy doody christmas song for the world is a little girl
hello again and the world is a little girl in my
hola mi vida es el amor de la vie en ti
gutentag 
ciao bella notte di te quiero ser feliz navidad negra tomasa
bonjour monsieur bibendum 
ola marina gasolina 
hej chlopcze 
